content/influxdb/v2.0/reference/cli/influxd/upgrade.md

```diff
@@ -49,6 +49,7 @@ influxd upgrade [command]
 |      | `--log-level`                   | Supported log levels are `debug`, `info`, `warn`, and `error` (default `info`)                             | string     |
 |      | `--log-path`                    | Custom log file path (default `~/upgrade.log`)                                                             | string     |
 | `-o` | `--org`                         | Primary organization name                                                                                  | string     |
+|      | `--overwrite-existing-v2`       | Overwrite existing files at the `engine-path` instead of aborting the upgrade process                      |            |
```

Comment:
More than just the `engine-path`. The full list is:
* `bolt-path`
* `continuous-query-export-path`
* `influx-configs-path`
* `engine-path`
* `v2-config-path`